backend/app/alembic/versions/2026_08_21_1358-1c57603aa0f6_drop_stale_storage_check_constraints.py

```
@@ -0,0 +1,35 @@
+"""drop_stale_storage_check_constraints
```

Ruff (0.16.1)

[warning] File `backend/app/alembic/versions/2026_08_21_1358-1c57603aa0f6_drop_stale_storage_check_constraints.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)